### PR TITLE
[PRM-528] - Added outstanding amount to LSP's

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

**Comment by LewisMitchell** [comment]
https://github.com/hmrc/penalties-frontend/pull/74
https://github.com/hmrc/penalties-acceptance-tests/pull/57

File: app/models/financial/Financial.scala [hunk 1]
```diff
@@ -22,6 +22,7 @@ import java.time.LocalDateTime
```
Comment (ruthdas03) [inline app/models/financial/Financial.scala:22]: Financial and PaymentFinancial models are the same , so there's a scope for optimisation (Stands out similar to Communication which is used in LatePaymentPenalty and PenaltyPoint)

Comment (LewisMitchell) [reply app/models/financial/Financial.scala:22]: Yeah I see! Good spot, will merge them into one model 😄

Comment (LewisMitchell) [reply app/models/financial/Financial.scala:22]: i'll need to update this on the frontend as well 👍

**Review by ruthdas03 — APPROVED** [review]

**Pull request merged**